packages/vite-plugin-svelte-inspector/types/index.d.ts

```diff
@@ -0,0 +1,74 @@
+declare module '@sveltejs/vite-plugin-svelte-inspector' {
+	export interface Options {
+		/**
+		 * define a key combo to toggle inspector,
+		 * @default 'meta-shift' on mac, 'control-shift' on other os
+		 *
+		 * any number of modifiers `control` `shift` `alt` `meta` followed by zero or one regular key, separated by -
+		 * examples: control-shift, control-o, control-alt-s  meta-x control-meta
+		 * Some keys have native behavior (e.g. alt-s opens history menu on firefox).
+		 * To avoid conflicts or accidentally typing into inputs, modifier only combinations are recommended.
+		 */
+		toggleKeyCombo?: string;
+
+		/**
+		 * define keys to select elements with via keyboard
+		 * @default {parent: 'ArrowUp', child: 'ArrowDown', next: 'ArrowRight', prev: 'ArrowLeft' }
+		 *
+		 * improves accessibility and also helps when you want to select elements that do not have a hoverable surface area
+		 * due to tight wrapping
+		 *
+		 * A note for users of screen-readers:
+		 * If you are using arrow keys to navigate the page itself, change the navKeys to avoid conflicts.
+		 * e.g. navKeys: {parent: 'w', prev: 'a', child: 's', next: 'd'}
+		 *
+		 *
+		 * parent: select closest parent
+		 * child: select first child (or grandchild)
+		 * next: next sibling (or parent if no next sibling exists)
+		 * prev: previous sibling (or parent if no prev sibling exists)
+		 */
+		navKeys?: { parent: string; child: string; next: string; prev: string };
+
+		/**
+		 * define key to open the editor for the currently selected dom node
+		 *
+		 * @default 'Enter'
+		 */
+		openKey?: string;
+
+		/**
+		 * inspector is automatically disabled when releasing toggleKeyCombo after holding it for a longpress
+		 * @default true
+		 */
+		holdMode?: boolean;
+
+		/**
+		 * when to show the toggle button
+		 * @default 'active'
+		 */
+		showToggleButton?: 'always' | 'active' | 'never';
+
+		/**
+		 * where to display the toggle button
+		 * @default top-right
+		 */
+		toggleButtonPos?: 'top-right' | 'top-left' | 'bottom-right' | 'bottom-left';
+
+		/**
+		 * inject custom styles when inspector is active
+		 */
+		customStyles?: boolean;
+
+		/**
+		 * internal options that are automatically set, not to be set or used by users
+		 * */
+		__internal?: {
+			// vite base url
+			base: string;
+		};
```

Comment:
This isn't marked `@internal` by dts-buddy. Perhaps not a big deal but figured to flag it here. Original:

https://github.com/sveltejs/vite-plugin-svelte/blob/24a14e4d6fe0758d619442cc62efcb24416c1bd4/packages/vite-plugin-svelte-inspector/src/public.d.ts#L62-L69

Reply:
noted https://github.com/Rich-Harris/dts-buddy/issues/63